### PR TITLE
01

### PR DESCRIPTION
**Pull request by faikp**

### CODE REVIEW

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
Try to group similar properties.
LGTM.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    justify-content: center;
+    align-items: center;
+}
+.container a{
+    position: relative;
+    width: 160px;
+    height: 60px;
+    display: inline-block;
+    background: #fff;
+    margin: 20px;
```
Comment (Rehmatulla-Khan) [inline style.css]: group similar properties so it will be more easy to read @faikp

File: style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    background: #0e1538;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    font-size: 1.2em;
+    color: #fff;
+    letter-spacing: 2px;
+    border: 2px solid #040a29;
+    overflow: hidden;
```
Comment (Rehmatulla-Khan) [inline style.css]: same here group the similar properties @faikp

**Review by Rehmatulla-Khan — APPROVED** [review]

File: style.css (hunk 2 above)
Comment (faikp) [reply style.css]: Yes Bhai , I'll do that changes👍🏻

**Review by mujeeb100 — COMMENTED** [review]
Nice!

**Comment by Rehan-Khan-Engineer** [comment]
LGTM

**Review by Rehan-Khan-Engineer — APPROVED** [review]

**Review by anask601 — APPROVED** [review]
Amazing @faikp

**Pull request merged**